pkg/controller/migplan/validation.go

```diff
@@ -683,10 +694,25 @@ func (r ReconcileMigPlan) validateSourceNamespaces(plan *migapi.MigPlan) error {
 // Validate required namespaces on the destination cluster.
 // Returns error and the total error conditions set.
 func (r ReconcileMigPlan) validateDestinationNamespaces(plan *migapi.MigPlan) error {
-	namespaces := []string{migapi.VeleroNamespace}
 	if plan.Status.HasAnyCondition(Suspended) {
 		return nil
 	}
+
+	namespaces := plan.GetDestinationNamespaces()
+	duplicates := listDuplicateNamespaces(namespaces)
+	if len(duplicates) > 0 {
+		plan.Status.SetCondition(migapi.Condition{
+			Type:     DuplicateNsOnDestinationCluster,
+			Status:   True,
+			Reason:   DuplicateNs,
+			Category: Critical,
+			Message:  "Duplicate destination cluster namespaces [] in migplan.",
```

Comment:
could we change "migplan" here to migration plan?

Reply:
Sure. I was thinking that API-level errors were better using the actual api resource names, but if we're using "migration plan" elsewhere I can do that here.

Reply:
Ahh. We're inconsistent here. It looks liike we *do* use `migstorage` and `migcluster` in other condition messages, but we also use `migration plan` elsewhere.

Reply:
Yeah, that was my conclusion as well. It could probably go either way to be honest.